.github/workflows/iterate_snowflake_ci.yml

```diff
@@ -0,0 +1,26 @@
+##################################################
+###   Visit https://github.com/tinybirdco/ci   ###
+###   for more details or custom CI/CD         ###
+##################################################
+
+name: Tinybird - CI Workflow
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'iterate_snowflake/**'
+    branches:
+      - main
+    types: [opened, reopened, labeled, unlabeled, synchronize, closed]
+
+concurrency: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+
+jobs:
+  ci:
+    uses: tinybirdco/ci/.github/workflows/ci.yml@v4.1.0
+    with:
+      data_project_dir: ./add_column_snowflake_ds
```

Comment:
If you look at the CI there's this warning:

```
** UPDATE AVAILABLE: please run "pip install tinybird-cli==5.7.0" to update or `export TB_VERSION_WARNING=0` to skip the check.
** current: tinybird-cli 5.0.0.dev0
```

The reason is `data_project_dir` does not point to `iterate_snowflake` which contains the requirements.txt. Same for the CD template